test/test_02_hw_09.py

```diff
@@ -0,0 +1,25 @@
+from hw.hw_09_task_02 import SuppressorCls, suppressor_gen
+
+import pytest
+
+
+def test_class_suppressed_passed_exception():
+    with SuppressorCls(IndexError):
+        [][2]
```

Comment:
What does this code do?

It raises IndexError.

Can we read Python code as an English phrase?  Yes! 
 
```python
raise IndexError()
```